nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/InvokeHTTP.java

```diff
@@ -1012,63 +981,59 @@ public void onTrigger(ProcessContext context, ProcessSession session) throws Pro
 
 
             // cleanup response flowfile, if applicable
-            try {
-                if (responseFlowFile != null) {
-                    session.remove(responseFlowFile);
-                }
-            } catch (final Exception e1) {
-                logger.error("Could not cleanup response flowfile due to exception: {}", new Object[]{e1}, e1);
```

Comment:
Is it safe to remove the try/catch?

Reply:
Since this try-catch block is wrapping a framework-level call to `session.remove()`, it seems unnecessary.  The `responseFlowFile` can be null, which is being checked.  However, if there is some other problem with the value of `responseFlowFile` at this point, that more likely points to a problem in the processor itself, should be corrected instead of just logged as an error.